### PR TITLE
Transaction Inclusion Proposal

### PR DESCRIPTION
**Pull request by carllin**
#### Problem
No way for light clients to independently verify a transaction was optimistically confirmed

#### Summary of Changes

Fixes #

### CODE REVIEW

**Comment by t-nelson** [comment]
Mind wrapping at 80 char to ease review? :pray:

File: sys-tuner/src/main.rs [hunk 1]
```diff
@@ -151,7 +151,8 @@ fn main() {
             info!("Tuning the system now");
             #[cfg(target_os = "linux")]
             {
-                // tune_poh_service_priority(peer_uid);
+                [cfg(target_os = "linux")]
```
Comment (ryoqun) [inline sys-tuner/src/main.rs:154]: @carllin oh again? ;)

Comment (t-nelson) [reply sys-tuner/src/main.rs]: https://github.com/rust-lang/rustfmt/issues/4452#issuecomment-705886282

**Review by t-nelson — APPROVED** [review]
Additions look good and jive with my mental model of what needs to happen.

One nit.  You may want to replace the GH links that target master with ones targeting a specific commit.  The linked code blocks will drift otherwise.

**Comment by carllin** [comment]
@t-nelson yup, updated!

**Pull request merged**